components/eam/src/chemistry/modal_aero/aero_model.F90

```diff
@@ -111,7 +111,7 @@ subroutine aero_model_readnl(nlfile)
 
     namelist /aerosol_nl/ aer_wetdep_list, aer_drydep_list, sol_facti_cloud_borne, seasalt_emis_scale,&
        sscav_tuning,sol_factb_interstitial, sol_factic_interstitial
-
+    ! FIXME: many of these namelists are not used, can be cleaned up.
     !-----------------------------------------------------------------------------
```

Comment:
Could we be more specific here? aer_wetdep_list and aer_drydep_list are for dry and wet dep output variable names' initialization. Probably C++ team has another way to do it. seasalt_emis_scale is a tunning factor for seasalt emission. sscav_tuning is always true? The other three sol_factxx are used in wetdep.

Reply:
Yes, I mean sscav_tuning, sol_factb_interstitial, and sol_factic_interstitial. sscav_tuning is always true, and the other two are removed in my refactored code.